File: Practice/30 Days of Code/Day 26 Nested Logic/Day 26 Nested Logic.java
import java.io.*;
import java.util.*;

public class Solution {
    static int getFine(int actualDay, int actualMonth, int actualYear, int expectedDay,      int expectedMonth, int expectedYear) {
        if(actualYear > expectedYear) return 10000;
        if(actualYear < expectedYear) return 0;
        if(actualMonth > expectedMonth) {
            return 500 * (actualMonth-expectedMonth);
        }
        if(actualMonth < expectedMonth) return 0;
        if(actualDay > expectedDay) {
            return 15 * (actualDay-expectedDay);
        }
        return 0;
    }

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner sc = new Scanner(System.in);
        int actualDay = sc.nextInt();
        int actualMonth = sc.nextInt();
        int actualYear = sc.nextInt();
        int expectedDay = sc.nextInt();
        int expectedMonth = sc.nextInt();
        int expectedYear = sc.nextInt();
        int fine = getFine(actualDay, actualMonth, actualYear, 
            expectedDay, expectedMonth, expectedYear);
        System.out.println(fine);
    }
}

File: Practice/30 Days of Code/Day 26 Nested Logic/Day 26 Nested Logic.php
<?php
function get_fine($actualDay, $actualMonth, $actualYear, $expectedDay, $expectedMonth, $expectedYear) {
    if($actualYear > $expectedYear) return 10000;
    if($actualYear < $expectedYear) return 0;
    if($actualMonth > $expectedMonth) {
        return 500*($actualMonth-$expectedMonth);
    }
    if($actualMonth < $expectedMonth) return 0;
    if($actualDay > $expectedDay) {
        return 15*($actualDay-$expectedDay);
    }
    return 0;
}
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */
fscanf($_fp, '%d %d %d', $actualDay, $actualMonth, $actualYear);
fscanf($_fp, '%d %d %d', $expectedDay, $expectedMonth, $expectedYear);
$fine = get_fine($actualDay, $actualMonth, $actualYear, $expectedDay, $expectedMonth, $expectedYear);
echo $fine;

?>
